tests/Monolog/LogProcessorTest.php

```diff
@@ -247,6 +248,44 @@ public function testProcessRecordExceptionBrokenUploader(): void
         self::assertEquals('', $newRecord['channel']);
     }
 
+    public function testProcessRecordDeprecationException(): void
+    {
+        $record = [
+            'message' => 'test exception',
+            'level' => Logger::CRITICAL,
+            'level_name' => 'CRITICAL',
+            'context' => [
+                'exceptionId' => '12345',
+                'exception' => new ErrorException('exception message', 543, E_USER_DEPRECATED),
+            ],
+        ];
+
+        $uploaderFactory = new UploaderFactory(
+            'https://example.com',
+            'runner-non-existent-bucket', // wrong configuration here to check if the uploader was triggered
```

Comment:
? spíš by to mělo otestovat, že je prázdnej attachment ne?

Reply:
upraveno